_docs-sources/guides/stay-up-to-date/patcher/index.md

```diff
@@ -0,0 +1,74 @@
+# Patcher
+
+## Installation
+
+To install Patcher, download the appropriate package for your system from the [releases](https://github.com/gruntwork-io/patcher-cli/releases) page.
+
+### For MacOS and Linux users
+
+Patcher runs in single binary called `patcher`. Make sure it's available within your `PATH`.
+
+1. After downloading Patcher, unzip the package (only MacOS releases are zipped). Patcher runs in a single binary.
+2. If necessary, rename the binary to `patcher`.
+```
+mv patcher_linux_amd64 patcher
+```
+3. Move `patcher` to a location available in your `PATH`.
+```
+echo $PATH
+mv patcher /usr/local/bin
+```
+
+## Before Running Patcher
+
+To fetch information from GitHub, Patcher requires a GitHub Personal Access Token, with the `repo` scope. Set the
+`GITHUB_OAUTH_TOKEN` environment variable to your GitHub Personal Access token:
+
+```bash
+export GITHUB_OAUTH_TOKEN="<YOUR_GITHUB_PAT>"
+```
+
+## Running Patcher
+
+Patcher should be run in a local Terraform or Terragrunt Git repo. It will analyze _all_ modules that belong to the current folder
+including its children. Patcher supports `source` values only from GitHub.
+
+In an `infrastructure-live` repository, we recommend running Patcher inside each environment folder, e.g. `infrastructure-live/dev`.
+
+### Patcher Report
+
+In `0.2.x`, the only available command is `patcher report`. This is a read-only version of Patcher that shows the changelog per module and its usages.
```

Comment:
```suggestion
In `v0.2.x`, the only available command is `patcher report`. This is a read-only version of Patcher that shows the changelog per module and its usages.
```

Reply:
It may also be helpful to say something like:

"After running `patcher report` you will end up in the 'Modules View'".

This context will be helpful in your next 2 sections where you refer to other command that are available from the 'Modules View'